test/test_rospy/CMakeLists.txt

```diff
@@ -50,6 +50,7 @@ if(CATKIN_ENABLE_TESTING)
   add_rostest(test/rostest/services.test)
   add_rostest(test/rostest/deregister.test)
   add_rostest(test/rostest/pubsub-order.test)
+  add_rostest(test/rostest/rosout.test)
```

Comment:
alphasort, unless there is a particular reason?

Reply:
I didn't want to change so many lines, but now I've done it.

Reply:
Please revert unrelated / unnecessary changes like this. They make backporting patches much more difficult for no reason.

Reply:
Done. Can you please review this pull request, @dirk-thomas?